### ISSUE TITLE
Opened Notebook file shows no content

### ISSUE THREAD

**Issue by diddlys** [microsoft/azuredatastudio#21588]
Issue Type: <b>Bug</b>

I have a .ipynb Notebook file that I've been using. Restarted my laptop overnight, reopened Data Studio with the Notebook file still open - and I can't see any of the contents?!

I have closed the file, re-opened it and still nothing showing. I *can* open the Notebook in VS Code and it shows up. 

To fix this, I opened the Notebook file in VS Code, made a small edit and saved it. It then opened in Data Studio. 

Azure Data Studio version: azuredatastudio 1.40.2 (661384637db384fe5d4e6224069adbe708580b16, 2022-12-22T22:17:02.658Z)
OS version: Windows_NT x64 10.0.19045
Restricted Mode: No
Preview Features: Enabled

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-11390H @ 3.40GHz (8 x 2918)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.74GB (2.59GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
admin-pack|Mic|0.0.2
agent|Mic|0.49.0
dacpac|Mic|1.11.0
import|Mic|1.5.4
profiler|Mic|0.11.0
sql-prompt|Red|0.2.11
sql-search|Red|0.4.1

</details>

**Comment by ervinte** [comment]
Same thing here.

Version: 1.40.2
Commit: 661384637db384fe5d4e6224069adbe708580b16
Date: 2022-12-22T22:05:12.502Z (3 wks ago)
VS Code: 1.67.0
Electron: 17.4.11
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.14-electron.0
OS: Darwin x64 22.2.0

**Comment by Charles-Gagnon** [comment]
1. Do you still have the Notebook that this happened with? And if you do would you be willing to share it (you can e-mail it to me directly if you'd prefer not to upload publicly)
2. Did this happen multiple times/with multiple Notebooks or just once?
3. Did it continue to occur even after closing the Notebook tab and re-opening it in ADS? (before you made the edit in VS Code)

If this happens again, it would be helpful if you would open up the developer console (Help -> Toggle Developer Console), go to the `Console` tab and then copy the logs there and then paste them here for us to look at.

**Comment by diddlys** [comment]
In answer to your questions:

1. I don't have the file with the issue - after editing in VS Code it worked, and I did not save the original. If it happens again I'll be happy to share the 'faulty' file.
2. In this case it was only the one affected Notebook file. I was able to open other Notebook files.
3. Yes, even after closing the file and re-opening it would not show the contents. I also tried re-starting ADS. Only after editing in VS Code was the file 'fixed'.
